test/board_test.exs

```diff
@@ -22,7 +22,7 @@ defmodule BoardTest do
     board = Board.new
     coords = [{0, 2}, {1, 0}, {2, 1}, {1, 2}]
     token = :x
-    assert Board.fill(board, coords, token) == %{
+    assert Board.update(board, coords, token) == %{
```

Comment:
The name `update` is very generic. How about `Board.take_move`? Or something similar?